packages/terra-section-header/CHANGELOG.md

```diff
@@ -2,6 +2,14 @@
 
 ## Unreleased
 
+## 2.57.0 - (October 19, 2022)
```

Comment:
The version and date are updated automatically during release
```suggestion

```

Reply:
Removed in https://github.com/cerner/terra-core/pull/3672/commits/e86233043581cf343e8122718429918c02acb022